src/webgpu/shader/execution/expression/call/builtin/texture_utils.ts

```diff
@@ -1410,6 +1414,10 @@ export async function checkCallResults<T extends Dimensionality>(
         errs.push('  sample points:');
         errs.push(layoutTwoColumns(expectedSamplePoints, gotSamplePoints).join('\n'));
         errs.push('', '');
+
+        // This path is slow so if we took it, don't report the other errors. One is enough
+        // to fail the test.
+        break;
```

Comment:
Looks like this is inside `if(sampler)`?
I think it should be between line 1421 and 1422.
We can add comments to brackets to better visualize.
```
} // if (sampler)
} // if (bad)
```

Reply:
Yes, because that's the slow path. Reporting all failures without samplers is fast.

Reply:
I can add the comments though

Reply:
Ok. If it is intended to only break for the sampler part not for all bad part then it looks good. (Not really need to tweak the bracket comment)